docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -508,23 +514,23 @@ Deriving :math:`r_{SAM,ux}` using the following equations:
 
 #. :math:`r_{SAM,tx} = 1 / \text{time to recovery}_\text{effectively treated SAM}`
 
-#. :math:`1 / \text{time to recovery}_\text{overall SAM} =  1 / \text{time to recovery}_\text{effectively treated SAM} * C * E_{SAM} + 1 / \text{time to recovery}_\text{untreated SAM} * (1 - C * E_{SAM})`
+#. :math:`1 / \text{time to recovery}_\text{overall SAM} =  1 / \text{time to recovery}_\text{effectively treated SAM} * C_{SAM} * E_{SAM} + 1 / \text{time to recovery}_\text{untreated SAM} * (1 - C_{SAM} * E_{SAM})`
```

Comment:
This equation goes off the right side of my screen...

Reply:
I split this up, thanks